### PR TITLE
Update portals.csv

### PR DESCRIPTION
**Pull request by opentechcommunity**
Added Open Data Nepal, Open Data portal

### CODE REVIEW

**Comment by opentechcommunity** [comment]
Added Open Data Nepal portal

File: data/portals.csv [hunk 1]
```diff
@@ -918,3 +918,4 @@ Transport for NSW is a state government agency in Australia.","transportation,lo
...
 portal-dados-urbanos-uff,Portal de Dados Urbanos PPGAU UFF - Open data portal for Urban Resarch,https://portaldados.ppgau.uff.br/,Plano C and PPGAU-UFF,Universidade Federal Fluminense,2022-01-03,Government,Open data portal customized for urbanism and urban studies researchers.,"brazil,academic,urbanism",cc,,License information on the portal is on a per dataset basis. Most datasets have no licensing information.,"RJ, Brazil","-22.90501, -43.13053",BR,pt,active,2022-06-19T23:58Z,CKAN,https://portaldados.ppgau.uff.br/api/3,CKAN API,
+open-data-nepal,"Open Data Nepal",https://opendatanepal.com/,Open Knowledge Nepal,Open Knowledge Nepal,,,"A civil society initiative to make Nepal’s data accessible online perpetually. The portal allows users to publish, explore, download and use data to make their own visualisations, apps and services without any restriction.",ckan nepal,cc-by,https://creativecommons.org/licenses/by/4.0/,,"Kathmandu, Nepal","27.7097,85.3209",NP,np,active,,ckan,https://opendatanepal.com/api,CKAN API,
```
Comment (augusto-herrmann) [inline data/portals.csv:921]:
```suggestion
open-data-nepal,"Open Data Nepal",https://opendatanepal.com/,Open Knowledge Nepal,Open Knowledge Nepal,,Community,"A civil society initiative to make Nepal’s data accessible online perpetually. The portal allows users to publish, explore, download and use data to make their own visualisations, apps and services without any restriction.",ckan nepal,cc-by,https://creativecommons.org/licenses/by/4.0/,,"Kathmandu, Nepal","27.7097,85.3209",NP,np,active,,ckan,https://opendatanepal.com/api,CKAN API,
```

If this is a community effort, we should probably use the publisher classification "Comminity" (see issue #271).

Comment (augusto-herrmann) [inline data/portals.csv:921]:
```suggestion
open-data-nepal,"Open Data Nepal",https://opendatanepal.com/,Open Knowledge Nepal,Open Knowledge Nepal,,,"A civil society initiative to make Nepal’s data accessible online perpetually. The portal allows users to publish, explore, download and use data to make their own visualisations, apps and services without any restriction.",ckan nepal,cc-by,https://creativecommons.org/licenses/by/4.0/,,"Kathmandu, Nepal","27.7097,85.3209",NP,en,active,,ckan,https://opendatanepal.com/api,CKAN API,
```

The data says the portal is in Nepalese, but it appears to be in English language.

Comment (augusto-herrmann) [reply data/portals.csv:921]: Also, do you happen to know the data when the portal launched? If so, that could go into the `issued` field.

**Review by augusto-herrmann — COMMENTED** [review]
Thanks for adding the portal with a PR!

@nikeshbalami, care to take a look, too?

File: data/portals.csv (hunk 1 above)
Comment (opentechcommunity) [reply data/portals.csv:921]: Oh! Got it. Didn't know that.

**Comment by nikeshbalami** [comment]
Thank you @opentechcommunity.

@augusto-herrmann this looks good. It was launched in "**2018**" (https://blog.okfn.org/2018/03/05/beta-version-of-open-data-nepal-a-portal-to-make-nepals-data-accessible-online/), lets add it and merge.

**Pull request merged**